sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -0,0 +1,74 @@
+import { EyeOpenIcon as EyeOpenIconBase } from '@radix-ui/react-icons';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { styled } from '../../../stitches.config';
+import { Button as ButtonBase, Container, Flex, Text } from '../../../ui';
+import { useAuth } from '../../auth/AuthContext';
+import { Logo } from './Logo';
+
+const Header = styled('header', Container, {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  mt: '$10',
+  mb: '$20',
+});
+
+const Nav = styled('nav', Flex, {
+  gap: '$10',
+});
+
+const EyeOpenIcon = styled(EyeOpenIconBase, {
+  display: 'inline-block',
+});
+
+const StatusDot = styled('div', {
+  backgroundColor: '#37C391',
+  width: '$2',
+  height: '$2',
+  borderRadius: '$round',
+});
+
+const Button = styled('a', ButtonBase);
```

Comment:
When we don't add any style, I tend to prefer using the `as` property to change the element easily, also make it more clear that the button is a link by just reading the JSX part of the file.

```
<Button as={'a'}>
 ....
</Button>
```

In the same idea, line 17 for the nav could be simplified to:

```
<Flex as="nav" gap="10">
 ....
</Flex>
```

Wdyt of this approach?

Reply:
Yes after your explanation and seeing the examples, it definitely makes more sense to use this approach!